srcpkgs/pipewire/INSTALL.msg

```diff
@@ -0,0 +1,5 @@
+Support for pipewire-media-session has been dropped by the pipewire developers,
+and will no longer be included in Void packages of pipewire after 0.3.64. 
+Wireplumber, an alternate session manager, is supported and packaged. If you 
+are currently using pipewire-media-session, you must install wireplumber and 
+change your configuration to use it before the next pipewire release.
```

Comment:
The pipewire-media-session session manager has been abandoned upstream and will be removed from Void in a subsequent pipewire update. All users are encouraged to transition to wireplumber immediately to avoid unexpected loss of functionality in the future.

Reply:
Changed to this wording.